src/class-parsely.php

```diff
@@ -416,8 +459,7 @@ public function api_key_is_set(): bool {
 		$options = $this->get_options();
 
 		return (
-				isset( $options['apikey'] ) &&
-				is_string( $options['apikey'] ) &&
+				is_string( $options['apikey'] ) && // @phpstan-ignore-line.
```

Comment:
We can remove this condition because `apikey` will always be string here .... but just keeping it for safety purpose i.e. for edge cases where we have some other type in the database.

Let me know if you think it is good to remove.

Reply:
I think that we can remove this, but have no issue to leave it as is for extra safety. I'll leave the final decision to you.

Reply:
Okay removing this then since user can only give this input from settings which indicates only strings can enter.